apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/stu3/providers/HHAClaimTransformerTest.java

```diff
@@ -60,13 +59,15 @@ public void transformSampleARecord() throws FHIRException, IOException {
    * @param claim the {@link HHAClaim} that the {@link ExplanationOfBenefit} was generated from
    * @param eob the {@link ExplanationOfBenefit} that was generated from the specified {@link
    *     HHAClaim}
-   * @param transformerContext the {@link TransformerContext} that was generated from the specified
-   *     {@link HHAClaim}
    * @throws FHIRException (indicates test failure)
    */
-  static void assertMatches(
-      HHAClaim claim, ExplanationOfBenefit eob, TransformerContext transformerContext)
-      throws FHIRException {
+  public static void assertMatches(HHAClaim claim, ExplanationOfBenefit eob) throws FHIRException {
+    /*
+     * Unfortunately this method is called from outside this class; need to create
+     * a NPIOrgLookup object for this verfication test.
+     */
+    NPIOrgLookup npiOrgLookup = new NPIOrgLookup();
```

Comment:
This should be a mock; we should have either an E2E, IT, or a focused test to test the NPIOrgLookup operates correctly, not be testing it as part of the claim transformer.

Reply:
I only wish that were the case....ever stu3 transformer unit test has a static method that is invoked from within various integration tests...thus mocking NPI and/or FDA drug display blows up when the unit test static method is invoked from the IT.

Reply:
ditto on ITs calling into unit test(s).

Reply:
Deferred to https://jira.cms.gov/browse/BFD-2766